ingest/src/main/java/com/connexta/ingest/exceptions/StoreException.java

```diff
@@ -6,9 +6,9 @@
  */
 package com.connexta.ingest.exceptions;
 
-public class StoreException extends Exception {
+public class StoreException extends IngestException {
 
-  public StoreException(String message, Throwable cause) {
-    super(message, cause);
+  public StoreException(Throwable cause) {
+    super("Store exception", cause);
```

Comment:
✏️ This seems weird to me as this message is not adding any value beyond the class itself. It is standard to let the client pass its message which would be more context specific or at least some info that is used to be a standard message that is also context specific.